cpp/tests/test_odesecir.cpp

```diff
@@ -477,22 +470,23 @@ TEST(TestOdeSecir, testSettersAndGetters)
     model.parameters.get<mio::osecir::CriticalPerSevere>()[(mio::AgeGroup)0]                = vec[18];
     model.parameters.get<mio::osecir::DeathsPerCritical>()[(mio::AgeGroup)0]                = vec[19];
 
-    EXPECT_NE(model.parameters.get<mio::osecir::IncubationTime>()[(mio::AgeGroup)0].get_distribution().get(), nullptr);
+    EXPECT_NE(model.parameters.get<mio::osecir::TimeExposed>()[(mio::AgeGroup)0].get_distribution().get(), nullptr);
```

Comment:
Dont understand why we need this line +the one in 481 because the parameter is not changed in between. Remove one of them?